src/extensions.ts

```diff
@@ -15,6 +15,7 @@ export * from './tour/extensions';
 export const EXTENSION_POINT_TDP_SCORE = 'tdpScore';
 export const EXTENSION_POINT_TDP_SCORE_IMPL = 'tdpScoreImpl';
 export const EXTENSION_POINT_TDP_SCORE_LOADER = 'tdpScoreLoader';
+export const EXTENSION_POINT_TDP_SCORE_COLUMN_PATCHER = 'tdpScoreColumnPatcher';
```

Comment:
I would use the new extension point id notation `EP_TDP_CORE_XXXX = 'epTdpCoreXXXXXXX'` (see [further down](https://github.com/datavisyn/tdp_core/pull/234/files#diff-887a9ff607524fd9a4e2cddfbec834c6R37)).

```suggestion
export const EP_TDP_CORE_SCORE_COLUMN_PATCHER = 'epTdpCoreScoreColumnPatcher';
```

Reply:
Maybe we should group the parts of extension points so that it becomes more clear which interfaces belong to which constant:

```ts
export const EXTENSION_POINT_TDP_SCORE_COLUMN_PATCHER = 'tdpScoreColumnPatcher';

export interface IScoreColumnPatcherExtension {
  factory(pluginDesc: IPluginDesc, colDesc: IAdditionalColumnDesc, rows: IScoreRow<any>[], col: Column): Promise<void>;
}

export interface IScoreColumnPatcherExtensionDesc extends IPluginDesc {
  load(): Promise<IPlugin & IScoreColumnPatcherExtension>;
}
```

Reply:
Please add a comment what the extension point does. You don't have to add comments about the `@registryParam` and `@factoryParam` as you described it as interface below.